routes/shopItemByStatus.js

```diff
@@ -1,25 +1,51 @@
 const express = require("express");
 const utility = require("../utility.js");
+const db = require("../server.js");
 
 findStatusRouter = express.Router();
 
 findStatusRouter.use(utility.readDatabaseMiddleware);
 
+//The original code
+// findStatusRouter.get("/", (req, res, next) => {
+//     const isCompleted = req.query.isCompleted || "";
+//     let groceryArray = req.database.shopItems;
+//     if (isCompleted === "true" || isCompleted === "false" || isCompleted === "") {
+//         if (isCompleted === "true") {
+//             groceryArray = utility.filterByStatus(groceryArray, true)
+//         } else if (isCompleted === "false") {
+//             groceryArray = utility.filterByStatus(groceryArray, false)
+//         }
+//         res.status(200);
+//         res.send(groceryArray);
+//     } else {
+//         res.status(400).send();
+//     }
+
+// });
```

Comment:
You can just remove this. You can always go back in the git history to see the original code.